lib/cdo/contact_rollups.rb

```diff
@@ -236,10 +239,18 @@ def self.insert_from_pegasus_forms
     start = Time.now
     log "Inserting contacts and IP geo data from pegasus.forms"
     PEGASUS_REPORTING_DB_WRITER.run "
-    INSERT IGNORE INTO #{PEGASUS_DB_NAME}.#{DEST_TABLE_NAME} (email, name, city, state, postal_code, country)
-    SELECT email, name, city, state, postal_code, country FROM #{PEGASUS_DB_NAME}.forms
+    INSERT INTO #{PEGASUS_DB_NAME}.#{DEST_TABLE_NAME} (email, name, roles, forms_submitted, city, state, postal_code, country)
+    SELECT email, name, 'Form Submitter', kind, city, state, postal_code, country FROM #{PEGASUS_DB_NAME}.forms
```

Comment:
That is the chosen role name for this to put in rollup, it will correspond to a role that shows up in Pardot. I realize now that this should be a Ruby constant, have pulled that and 'Teacher' into constants.